src/Sparrow/Json/BlittableJsonReaderObject.cs

```diff
@@ -1303,7 +1303,7 @@ public static void AssertNoModifications(BlittableJsonReaderObject data, string
             }
         }
 
-        public bool Contains(LazyStringValue propertyName)
+        public bool Contains(string propertyName)
```

Comment:
This will materialize a lot of LazyStringValues, can we avoid that. I do not see you using that anywhere?

Reply:
Yes, this is to be avoided if possible.

Reply:
done